ooniapi/services/ooniprobe/tests/integ/test_geolookup.py

```diff
@@ -0,0 +1,12 @@
+def test_geolookup(client):
+    j = dict(
+        addresses=["192.33.4.12", "170.247.170.2", "2801:1b8:10::b", "2001:500:2::c"]
+    )
+    c = client.post("/api/v1/geolookup", json=j).json()
+    assert "geolocation" in c
+    assert "v" in c
+    g = c["geolocation"]
+    for ip in j["addresses"]:
+        assert g[ip]["cc"] != "ZZ"
```

Comment:
To have a control test that won't rely on the ip database I would add another one that monkey patches the `lookup_probe_network` and `lookup_probe_cc` functions  to lookup from a hard coded dictionary

Example: https://github.com/ooni/backend/blob/5763024ff0859fa335971080b99f6ae89766160f/ooniapi/services/oonimeasurements/tests/test_measurements.py#L167